resources/osl_gitlfs.rb

```diff
@@ -0,0 +1,21 @@
+resource_name :osl_gitlfs
+
+default_action :sync
+
+provides :osl_gitlfs
+property :directory, String,  required: true
+property :repository, String, required: true
```

Comment:
Which of these are the name property? I'm thinking probably directory. You need to set that.